stdlib/public/runtime/CygwinPort.cpp

```diff
@@ -47,7 +47,7 @@ int swift::_swift_dl_iterate_phdr(int (*Callback)(struct dl_phdr_info *info,
   BOOL ret = EnumProcessModules(procHandle, modules.data(),
                                 modules.size() * sizeof(HMODULE), &neededSize);
 
-  if (ret == FALSE) {
```

Comment:
I naïvely assumed `0` but I guess we should be on the safe side and check with the Windows/Cygwin guru @tinysun212 :-)

@tinysun212, do you have time to review? That would be appreciated! :-)